spreadsheets.js

```diff
@@ -422,8 +438,103 @@ function update_schedule(service_account_auth, key, colname, format, schedule, c
     });
 }
 
+// parse the input string for a results update
+function parse_result(input_string){
```

Comment:
I like the way you organized this code. if I find the time, I will update mine to look more like this as well.